lib/mumukit/isolated_environment.rb

```diff
@@ -13,7 +13,8 @@ def configure!(*files)
       binds = dirnames.map { |it| "#{it}:#{it}" }
       volumes = Hash[dirnames.map { |it| [[it, {}]] }]
 
-      command = yield(*filenames).split
+      command = yield(*filenames)
+      command = command.split if String === command
```

Comment:
Why is this necessary?

Reply:
It gives us a little more control over the command to execute. For example, say we need to pass something through a command by piping it, if we allow only a string as a command, then something like `bash -c "echo something | command"` would be split to `['bash', '-c', '"echo', 'something', '|', 'command']` instead of the correct form: `['bash', '-c', 'echo something | command']`